test-spicy-features

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env bash
+
+BASE=$(cd $(dirname $0) && pwd)
+
+TMP=/tmp/$(basename $0).$$
+DIAG=${TMP}/diag.log
+OUTPUT=${TMP}/output.log
+
+#trap 'rm -rf ${TMP}' EXIT
+
+mkdir ${TMP}
+cd ${TMP}
+
+for i in zeek spicyz; do
+    if ! command -v $i >/dev/null; then
+        echo "no $i in PATH" >&2
+        exit 1
+    fi
+done
+
+function build_and_test {
+    rm -rf spicy-test
+
+    ZKG_DEFAULT_TEMPLATE=${BASE} zkg create --features=$1 --packagedir spicy-test
```

Comment:
Let's instead of using stdin specify the full arg list here,
```suggestion
    ZKG_DEFAULT_TEMPLATE=${BASE} zkg create --packagedir spicy-test "$@"
```

The call sites then become e.g.,

```shell
# Packet analyzer
build_and_test \
    --features=spicy-packet-analyzer \
    --user-var name=MyPackageX \
    --user-var analyzer=MyAnalyzerX \
    --user-var unit=MyPacketX
```

The cases where the default responder for the protocol analyzer is used should include

```
    --user-var unit_resp=
```